### ISSUE TITLE
v2.3.3 Can't fetch channel videos 

### ISSUE THREAD

**Issue by glesperance** [LuanRT/YouTube.js#235]
### Steps to reproduce

```
/**
 * Fetches all the videos for a channel, going through all the pages,
 * of results if necessary.
 * @param channelId The channel ID to fetch videos for.
 */
export async function listChannelVideos(
  channelId: string,
): Promise<GridVideo[]> {
  const youtube = await Innertube.create();
  const channel = await youtube.getChannel(channelId);
  const videos = await channel.getVideos();

  const result = [...videos.videos];

  console.log("@@ videos", videos.videos.length);

  if (videos.has_continuation) {
    let feed;
    do {
      feed = await (feed || videos).getContinuation();
      result.push(...feed.videos);
    } while (feed.has_continuation);
  }
  return result as GridVideo[];
}
```

### Failure Logs

```shell
console.warn
    InnertubeError: Something went wrong at Video!
    This is a bug, please report it at https://github.com/LuanRT/YouTube.js/issues
        at Function._Parser_printError (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:482:7)
        at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:382:27)
        at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:430:39)
        at new RichItem (/Workspace/project/node_modules/youtubei.js/src/parser/classes/RichItem.ts:12:27)
        at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)
        at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:417:29)
        at new RichGrid (/Workspace/project/node_modules/youtubei.js/src/parser/classes/RichGrid.ts:16:28)
        at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)
        at new Tab (/Workspace/project/node_modules/youtubei.js/src/parser/classes/Tab.ts:22:27)
        at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)
        at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:417:29)
        at new TwoColumnBrowseResults (/Workspace/project/node_modules/youtubei.js/src/parser/classes/TwoColumnBrowseResults.ts:12:24)
        at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)
        at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:430:39)
        at Function.parseResponse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:219:29)
        at new Feed (/Workspace/project/node_modules/youtubei.js/src/core/Feed.ts:43:27)
        at new TabbedFeed (/Workspace/project/node_modules/youtubei.js/src/core/TabbedFeed.ts:11:5)
        at Channel.<anonymous> (/Workspace/project/node_modules/youtubei.js/src/core/TabbedFeed.ts:34:12)
        at Generator.next (<anonymous>)
        at fulfilled (/Workspace/project/node_modules/youtubei.js/dist/src/core/TabbedFeed.js:5:58)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (node:internal/process/task_queues:96:5) {
      info: {
        stack: "TypeError: Cannot read properties of undefined (reading 'runs')\n" +
          '    at new NavigatableText (/Workspace/project/node_modules/youtubei.js/src/parser/classes/misc/NavigatableText.ts:13:12)\n' +
          '    at new Author (/Workspace/project/node_modules/youtubei.js/src/parser/classes/misc/Author.ts:21:22)\n' +
          '    at new Video (/Workspace/project/node_modules/youtubei.js/src/parser/classes/Video.ts:59:19)\n' +
          '    at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:430:39)\n' +
          '    at new RichItem (/Workspace/project/node_modules/youtubei.js/src/parser/classes/RichItem.ts:12:27)\n' +
          '    at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:417:29)\n' +
          '    at new RichGrid (/Workspace/project/node_modules/youtubei.js/src/parser/classes/RichGrid.ts:16:28)\n' +
          '    at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)\n' +
          '    at new Tab (/Workspace/project/node_modules/youtubei.js/src/parser/classes/Tab.ts:22:27)\n' +
          '    at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:417:29)\n' +
          '    at new TwoColumnBrowseResults (/Workspace/project/node_modules/youtubei.js/src/parser/classes/TwoColumnBrowseResults.ts:12:24)\n' +
          '    at Function.parseItem (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:430:39)\n' +
          '    at Function.parseResponse (/Workspace/project/node_modules/youtubei.js/src/parser/index.ts:219:29)\n' +
          '    at new Feed (/Workspace/project/node_modules/youtubei.js/src/core/Feed.ts:43:27)\n' +
          '    at new TabbedFeed (/Workspace/project/node_modules/youtubei.js/src/core/TabbedFeed.ts:11:5)\n' +
          '    at Channel.<anonymous> (/Workspace/project/node_modules/youtubei.js/src/core/TabbedFeed.ts:34:12)\n' +
          '    at Generator.next (<anonymous>)\n' +
          '    at fulfilled (/Workspace/project/node_modules/youtubei.js/dist/src/core/TabbedFeed.js:5:58)\n' +
          '    at runMicrotasks (<anonymous>)\n' +
          '    at processTicksAndRejections (node:internal/process/task_queues:96:5)'
      },
      date: 2022-11-12T17:13:17.838Z,
      version: '2.3.3'
    }
```

### Expected behavior

>  console.log("@@ videos", videos.videos.length);
Should print value > 0 -- and we shouldn't get a parse error.

### Current behavior

Right now it is impossible to fetch the videos of a channel.

### Version

Default

### Checklist

- [X] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.

**Comment by glesperance** [comment]
Stock example also breaks
https://github.com/LuanRT/YouTube.js/blob/main/examples/channel/basic-info.ts#L17-L22

**Comment by LuanRT** [comment]
This was fixed in #233. Will release a new version soon.

**Comment by glesperance** [comment]
Gotcha. Thanks @LuanRT
